### PR TITLE
Display error on UI in case of return error

### PR DESCRIPTION
**Pull request by jyvenugo**

### CODE REVIEW

File: preview/MsixCore/MsixCoreInstaller/InstallUI.cpp [hunk 1]
```diff
@@ -660,6 +693,23 @@ void UI::UpdateDisplayPercent(float displayPercent)
...
+    ShowWindow(g_staticPercentText, SW_HIDE);
+    ShowWindow(g_progressHWnd, SW_HIDE);
+    ShowWindow(g_checkboxHWnd, SW_HIDE);
+    ShowWindow(g_CancelbuttonHWnd, SW_HIDE);
+    
+    //Show Error Box
+    ShowWindow(g_staticErrorTextHWnd, SW_SHOW);
+    ShowWindow(g_staticErrorDescHWnd, SW_SHOW);
+    std::wstringstream errorDescription;
+    errorDescription << "App installation failed with error message: An internal error occurred with error " << hr << ".";
```
Comment (wcheng-msft) [inline preview/MsixCore/MsixCoreInstaller/InstallUI.cpp]: ><< [](start = 109, length = 2)

use std::hex to show this in hex instead of negative number

Comment (wcheng-msft) [inline preview/MsixCore/MsixCoreInstaller/InstallUI.cpp]: >"App installation failed with error message: An internal error occurred with error  [](start = 24, length = 83)

i think best to make this a string resource?

Comment (jyvenugo) [reply preview/MsixCore/MsixCoreInstaller/InstallUI.cpp]: Modified to use string resource.

Comment (jyvenugo) [reply preview/MsixCore/MsixCoreInstaller/InstallUI.cpp]: Modified to use std::hex.

**Pull request merged**